.rubocop.yml

```diff
@@ -13,6 +15,10 @@ Layout/SpaceAroundOperators:
 Layout/SpaceInsideHashLiteralBraces:
   EnforcedStyle: no_space
 
+Layout/AlignHash:
```

Comment:
Please put this at the before `Layout/DotPosition` (so that they are alphabetically sorted.

Reply:
Done and rebased.